swupd/packs_test.go

```diff
@@ -385,8 +409,12 @@ func mustValidateZeroPack(t *testing.T, manifestPath, packPath string) {
 		if err != nil {
 			t.Fatalf("error reading pack: %s", err)
 		}
-		if hdr.Name == "staged/" || !strings.HasPrefix(hdr.Name, "staged/") {
-			continue
+		if hdr.Name == "staged/" || hdr.Name == "delta/" {
+			t.Fatalf("multiple entries of %s directory", hdr.Name)
+		}
+
+		if !strings.HasPrefix(hdr.Name, "staged/") {
```

Comment:
And a comment that delta/ shouldn't have anything in it for a zero pack?

Reply:
Done.